### PR TITLE
MES-2046 - Implementing controlled stop selection

### PR DESCRIPTION
**Pull request by szabolcs-keresztes**
## Description and relevant Jira numbers

This PR adds the ability to mark a controlled stop as completed or not completed on the test report

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [x] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [ ] Reviewers selected in Github
- [x] Any new reusable component/widget added to component library on Confluence

- [x] Screenshot(s) captured on the physical device (if applicable)
- [x] Tested by QA
- [x] PO's approval

UNADJUSTEDNONRAW_thumb_50

UNADJUSTEDNONRAW_thumb_51

### CODE REVIEW

File: src/pages/test-report/components/controlled-stop/controlled-stop.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
+controlled-stop {
+
```
Comment (ammarhaiderbjss) [inline src/pages/test-report/components/controlled-stop/controlled-stop.scss]: is this all just copied over from competency? If so, can we used `@extend` instead?

File: src/pages/test-report/test-report.ts [hunk 2]
```diff
@@ -111,7 +110,9 @@ export class TestReportPage extends BasePageComponent {
       isRemoveFaultMode$.pipe(map(result => this.isRemoveFaultMode = result)),
       isSeriousMode$.pipe(map(result => this.isSeriousMode = result)),
       isDangerousMode$.pipe(map(result => this.isDangerousMode = result)),
-      manoeuvres$.pipe(map(manoeuvres => this.manoeuvresCompleted = Object.values(manoeuvres)[0])),
+
+      // TODO: [MES-2046] Not correct, controlled stop is not part of the manoeuvres
```
Comment (ammarhaiderbjss) [inline src/pages/test-report/test-report.ts]: Just for my benefit what is there to do here exactly?

Comment (Cavalier07) [reply src/pages/test-report/test-report.ts]: Nothing I forgot to remove the TODO - It's one szabi left for me yesterday

File: src/pages/test-report/components/controlled-stop/controlled-stop.scss (hunk 1 above)
Comment (Cavalier07) [reply src/pages/test-report/components/controlled-stop/controlled-stop.scss]: 90% of it is, theres a few changes in sizes. I think we need to refactor the css for all the buttons . Legal Requirements & ETA's also do the same , so we have pretty much the same css 4 times.

Are you happy for me to do this as a second PR?

Comment (ammarhaiderbjss) [reply src/pages/test-report/components/controlled-stop/controlled-stop.scss]: of course

**Pull request merged**